lib/teams/emr-on-eks-team.ts

```diff
@@ -58,7 +62,7 @@ export interface EmrEksTeamProps extends TeamProps {
  *It can either create a namespace or use an existing one
  *The class will set the necessary k8s RBAC needed by EMR on EKS as defined in the AWS documentation 
  * https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html
- * The class constructor take a the EMR on EKS Team definition
+ * The class constructor take an EMR on EKS Team definition and need the EMR on EKS AddOn added
```

Comment:
class constructor takes EMR on EKS team definition as a parameter. Pre:requisite: EMS on EKS AddOn is part of the blueprint.